test/controllers/sections/items/ItemProducerSizeControllerSpec.scala

```diff
@@ -159,19 +170,72 @@ class ItemProducerSizeControllerSpec extends SpecBase with MockPreDraftService w
         status(result) mustEqual SEE_OTHER
         redirectLocation(result).value mustEqual controllers.routes.JourneyRecoveryController.onPageLoad().url
       }
+    }
+
+    "for a GET unableToProvideInformation" - {
+
+      "must redirect to the next page" in new Setup(Some(
+        emptyUserAnswers.set(ItemExciseProductCodePage(0), testEpcWine).set(ItemProducerSizePage(testIndex1), BigInt(1))
```

Comment:
Personally, I don't like having `0` as an implicit for Index(0) but to make it consistent:
```suggestion
        emptyUserAnswers.set(ItemExciseProductCodePage(0), testEpcWine).set(ItemProducerSizePage(0), BigInt(1))
```